app/models/ability.rb

```diff
@@ -29,7 +29,8 @@ def initialize(user)
 
     # *** ADMINS and OWNERS ***
     can :manage, :all, company_id: user.company_id
-    cannot :destroy, User, role: User::ROLES[2]
+    cannot :destroy, User, company_id: user.company_id, role: User::ROLES[2]
```

Comment:
why is it required? "cannot"

Reply:
Buttons do not disappear when use only "can"

Reply:
There was one extra can. I have removed it.